packages/block-editor/src/components/list-view/block-select-button.js

```diff
@@ -83,8 +83,7 @@ function ListViewBlockSelectButton(
 
 	return (
 		<Button
-			// TODO: Switch to `true` (40px size) if possible
-			__next40pxDefaultSize={ false }
+			__next40pxDefaultSize
```

Comment:
**Before:**
![link-view-1-before](https://github.com/user-attachments/assets/1a37d042-cfd0-4e39-8f17-dc981a3d507f)


**After:**
![link-view-1-after](https://github.com/user-attachments/assets/3a54def9-b319-4992-9bde-5bf84fa21fa0)

Reply:
Could we consider removing the style override, and setting `size="compact"` instead?

Reply:
Unfortunately removing style overrides isn't trivial, since [they also affect the drop indicator in the list view](https://github.com/WordPress/gutenberg/blob/c56672504bada434aee1ec331366cd2a10ab2b38/packages/block-editor/src/components/list-view/drop-indicator.js#L318-L321).

I think that a better alternative is to get rid of `Button` altogether here, and instead replace it with a vanilla `<a />`

```diff
diff --git a/packages/block-editor/src/components/list-view/block-select-button.js b/packages/block-editor/src/components/list-view/block-select-button.js
index 9ea9d3b4a7..80847c44de 100644
--- a/packages/block-editor/src/components/list-view/block-select-button.js
+++ b/packages/block-editor/src/components/list-view/block-select-button.js
@@ -82,9 +82,7 @@ function ListViewBlockSelectButton(
 	}
 
 	return (
-		<Button
-			__next40pxDefaultSize
-			size="compact"
+		<a
 			className={ clsx(
 				'block-editor-list-view-block-select-button',
 				className
@@ -156,7 +154,7 @@ function ListViewBlockSelectButton(
 					</span>
 				) }
 			</HStack>
-		</Button>
+		</a>
 	);
 }
 
diff --git a/packages/block-editor/src/components/list-view/style.scss b/packages/block-editor/src/components/list-view/style.scss
index 3ad6d3bd4a..297fb9d0a6 100644
--- a/packages/block-editor/src/components/list-view/style.scss
+++ b/packages/block-editor/src/components/list-view/style.scss
@@ -216,6 +216,15 @@
 		text-align: left;
 		position: relative;
 		white-space: nowrap;
+		border-radius: 2px;
+		box-sizing: border-box;
+		color: inherit;
+		font-family: inherit;
+		font-size: 13px;
+		font-weight: 400;
+		margin: 0;
+		text-decoration: none;
+		transition: box-shadow 0.1s linear;
 
 		.components-modal__content & {
 			padding-left: 0;

```